keyboards/1upkeyboards/super16/keymaps/nblyumberg/keymap.c

```diff
@@ -0,0 +1,188 @@
+#include QMK_KEYBOARD_H
+//#define RBG_VAL 16 //Define 16 RGB LEDs
+
+uint16_t copy_paste_timer;
+uint16_t enter_timer;
+
+extern rgblight_config_t rgblight_config;
+
+// Define custom keycodes
+enum my_keycodes {
+	KC_CCCV = SAFE_RANGE,
+	KC_2ENTER
+};
+
+const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
+//Layer 0 - Base Layer (F13 to F24, and One Shot Layer 1,2,3,4)
+
+	[0] = LAYOUT_ortho_4x4(
+		KC_F13, KC_F14, KC_F15, KC_F16, 
+		KC_F17, KC_CCCV, KC_F19, KC_F20, //Changing KC_F18 to KC_CCCV 
+		KC_F21, KC_F22, KC_F23, KC_2ENTER, //Changing KC_F24 to KC_2ENTER
+		OSL(1), OSL(2), OSL(3), TG(4)  //One Shot Layer Activation
```

Comment:
Why not use one shot mods here, instead?

Reply:
I was working with the layers at the time so OSL made the most sense and it worked with AutoHotKey. I suppose it's just two different ways to accomplish the same thing.